model/src/main/proto/spine_examples/shareaware/investment/shares_purchase.proto

```diff
@@ -52,5 +52,8 @@ message SharesPurchase {
     ShareId share = 3 [(required) = true];
 
     // The quantity of shares to purchase.
-    int32 quantity = 4 [(required) = true];
+    int32 quantity = 4;
```

Comment:
I think it can cause some misunderstanding with the `shares_available` field.
Please, consider renaming.

Reply:
Maybe it will be better to name it `purchase_shares_quantity`? But it is a bit long.

@armiol @Vladyslav-Kuksiuk WDYT?

Reply:
I think, in this context it's perfectly fine: `SharesPurchase.quantity`.

Why would we need to repeat ourselves over and over?